### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JamesH65**
Add a IR filter transmission graph.

### CODE REVIEW

File: hardware/camera/README.md [hunk 1]
```diff
@@ -73,6 +73,12 @@ Full camera software documentation can be found [here]](../../raspbian/applicati
 | Triggers | Keypress, UNIX signal, timeout |
 | Extra modes | demo, burst/timelapse, circular buffer, video with motion vectors, segmented video, live preview on 3D models |
 
+## HQ Camera IR Filter Transmission Information
+
+The HQ Camera uses a Hoya CM500 Infrared filter. It's transmission characteristics are as represented in the following graph.
```
Comment (andrum99) [inline hardware/camera/README.md:78]: For consistency, the heading should not be capitalised after the first word (except for the part that is the product name).
Infrared -> infrared
it's -> its

File: hardware/camera/README.md [hunk 2]
```diff
@@ -73,6 +73,12 @@ Full camera software documentation can be found [here]](../../raspbian/applicati
...
+
+![GitHub Logo](./hoyacm500.png)
```
Comment (andrum99) [inline hardware/camera/README.md:80]: You probably want to change the image name tag 😋

Comment (JamesH65) [reply hardware/camera/README.md:80]: Weird - thought I had already changed that.

**Comment by lurch** [comment]
Pushed a typo fix to this PR

**Pull request merged**